main.go

```diff
@@ -72,4 +75,12 @@ func main() {
 
 	fmt.Println("Serving...")
 	http.ListenAndServe(":3841", nil)
+
+	// Initialize a test user named Steve
+	Steve := User{
+		Username: "steve",
+		Email: 		"steve@me.com",
+	}
+	h.initUser(&Steve)
```

Comment:
This line doesn't make much sense, there's no need to add a test user during initialization, this code runs every time you restart the server.